tests/performance_tests/performance_test_basic.py

```diff
@@ -70,6 +132,14 @@ def waitForEmptyBlocks(node):
     TestHelper.printSystemInfo("BEGIN")
     cluster.killall(allInstances=killAll)
     cluster.cleanup()
+
+    scriptName  = __file__.split("/")[-1][:-3]
```

Comment:
Since we've already imported `os`, prefer `os.path.splitext(os.path.basename(__file__))[0]`, which is portable across filesystems with other path separators and is more likely to behave well if the script has a strange name.  Admittedly neither scenario is likely to apply to this code any time soon but a habit of writing the more robust version is worth cultivating.

Reply:
Done.

Reply:
Addressed: f7d5ec955a5d6f367d245e9162d2e870468b3f93